lib/trim_decorator.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'base_decorator'
+
+class TrimDecorator < BaseDecorator
+  def correct_name
+    super()
+    @nameable.correct_name[0...10]
+  end
```

Comment:
The name has a maximum of 10 characters. If it's longer it should trim the word.

- Since indexes start at 0, the 1st character is at index 0 and the 10th character is at index 9. Therefore, your range `[0...10` returns 11 characters.
- Please include conditional statements to check if the length of the name is greater than 10.

i.e
```
correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
```

Reply:
@Danie12345 You are right.
You used three dots `...` which is not inclusive. Whereas my example uses two dots `..` which is inclusive. Both solutions are correct, my bad. 

- kindly ignore this requirement.

Reply:
@KDlamini alright, thank you. I'll take it this means it's approved?